roles/validate_operator_bundle/tasks/main.yml

```diff
@@ -28,10 +28,49 @@
     bundle_validate: "{{ operator_work_dir }}"
   when: not run_upstream|bool
 
-- name: "Set the operatorhub checks if enabled"
+- name: "list work_dir"
+  shell: "ls {{ work_dir }}"
+
+- name: "Get supported OCP versions"
+  block:
+    - name: Get skopeo inspect json
+      shell: "cat {{ work_dir }}/bundle-skopeo-inspect.json"
+      register: bundle_skopeo_inspect_json
+
+    - name: Convert json to map
+      set_fact:
+        bundle_skopeo_inspect_map: "{{ bundle_skopeo_inspect_json.stdout | from_json }}"
+
+    - name: Determine ocp version annotation
+      set_fact:
+        ocp_version_annotation: "{{ bundle_skopeo_inspect_map['Labels']['com.redhat.openshift.versions'] }}"
+
+    - name: "Query Pyxis"
+      uri:
+        url: "https://catalog.redhat.com/api/containers/v1/operators/indices?sort_by=data.ocp_version&ocp_versions_range={{ ocp_version_annotation }}&organization=redhat-operators"
+        method: GET
+        headers:
+          Accept: "application/json"
+        status_code: 200
```

Comment:
We need to somehow handle and do nice error report for invalid syntax
e.g. 
```
[alcik@localhost][/tmp/data]$ curl "https://catalog.redhat.com/api/containers/v1/operators/indices?sort_by=data.ocp_version&ocp_versions_range=v4.9,v4.10&organization=redhat-operators" | jq .
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   245  100   245    0     0    413      0 --:--:-- --:--:-- --:--:--   412
{
  "detail": "CSV filter is not supported for versions other than: (4.5,4.6,4.7,4.8) in any order. Unsupported version(s): (4.9,4.10)",
  "status": 400,
  "title": "Bad Request",
  "type": "about:blank",
  "trace_id": "0xffce4c92d42874d98e20ba0960cc59f4"
}
```

+ add code coverage for that